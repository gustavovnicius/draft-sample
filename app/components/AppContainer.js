import React from 'react'

const styles = {
  container: {
    display: 'flex',
    alignItems: 'center',
    flexDirection: 'column',
    padding: '3vh 0',
  }
}

const AppContainer = ({children}) => {
  return (
    <div style={styles.container}>
      {children}
    </div>
  )
}

export default AppContainer;
