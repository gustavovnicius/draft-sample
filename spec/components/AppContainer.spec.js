import React from 'react'
import renderer from 'react-test-renderer'
import AppContainer from 'components/AppContainer.js'

describe('AppContainer', () => {
  it('renders', () => {
    const app = renderer.create(
      <AppContainer>
        <div> Hello! </div>
      </AppContainer>
    );
    const tree = app.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
