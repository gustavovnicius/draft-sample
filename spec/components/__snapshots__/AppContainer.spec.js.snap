exports[`AppContainer renders 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "padding": "3vh 0",
    }
  }>
  <div>
     Hello! 
  </div>
</div>
`;
